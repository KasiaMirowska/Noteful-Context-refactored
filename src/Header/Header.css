header {
    background-color: rgb(54, 54, 107); 
    color: orange;
    height: 120px;
    font-family: 'Barriecito', cursive;
    text-align: left;
    
    font-size: 30px;
    /* padding-bottom: 10px; */
}

h1 {
    margin: 0;
    padding-left: 25%;
    padding-top: 20px;
}


