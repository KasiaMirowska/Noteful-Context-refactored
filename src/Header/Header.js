import React from 'react';
import './Header.css';

export default function Header() {
    return (
        <div>
            <header>
                <h1>NOTEFUL</h1>
            </header>
        </div>
    )
};