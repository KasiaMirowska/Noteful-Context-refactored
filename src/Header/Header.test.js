import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import Header from './Header';
import {BrowserRouter as BR} from 'react-router-dom';


describe('Header component', ()=>{
    it ('renders without crashing', ()=>{
        const div = document.createElement('div');
        ReactDom.render(<BR><Header /></BR>, div);
        ReactDom.unmountComponentAtNode(div);
    })
    it('renders UI as expected', () => {
        const header = renderer.create(<BR><Header /></BR>)
        expect(header.toJSON()).toMatchSnapshot();
    })
})