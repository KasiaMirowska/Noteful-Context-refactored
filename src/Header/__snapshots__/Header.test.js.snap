// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders UI as expected 1`] = `
<div
  className="header-container"
>
  <header>
    <h1>
      NOTEFUL
    </h1>
  </header>
</div>
`;
